projects/plugins/protect/src/js/components/vulnerabilities-list/list.jsx

```diff
@@ -29,8 +46,8 @@ const VulAccordionItem = ( { id, name, version, title, description, icon, fixedI
 				if ( ! [ 'core', 'plugin', 'theme' ].includes( type ) ) {
 					return;
 				}
-				recordEvent( `jetpack_protect_${ type }_vulnerability_open` );
-			}, [ recordEvent, type ] ) }
+				recordEventHandler( `jetpack_protect_${ type }_vulnerability_open` );
+			}, [ recordEventHandler, type ] ) }
```

Comment:
Nice use of `recordEventHandler` to handle both vulnerability open and security link clicks :+1: